checkbox-ng/plainbox/vendor/system_information/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
+
+INXI_PATH = str((Path(__file__).parent / "inxi").resolve())
```

Comment:
(python 3.5 does not support paths in subprocess calls, I dropped the type here rather than in the collector code to not write strange `str(INXI_PATH)` or wonkly `[str(x) for x in cmd]` like statements. Makes me sad as well to be honest. Either way we have to reconsider if we want to go this route, bundling inxi this way or not, the type thing we can fix once we decide)